GitUI/Editor/FileViewer.cs

```diff
@@ -1476,7 +1481,7 @@ public void ResetNoncommittedSelectedLines()
                 return;
             }
 
-            if (MessageBox.Show(this, TranslatedStrings.ResetSelectedLinesConfirmation, TranslatedStrings.ResetChangesCaption,
+            if (_unitTestConfirmResetLines && MessageBox.Show(this, TranslatedStrings.ResetSelectedLinesConfirmation, TranslatedStrings.ResetChangesCaption,
```

Comment:
We should change `MessageBox.Show` to `TaskDialog` and then we won't have test-only code outside the `TestAccessor`.

Reply:
Please have a look at `Use TaskDialog instead of MessageBox.Show for reset selected lines confirmation` commit. I've tried to look for examples of manipulating `TaskDialogPage` in the tests but couldn't. I am not sure if this is a correct approach.